cmake/qtifw/install_win_copydll.qs

```diff
@@ -52,11 +52,12 @@ function Component()
           // (=overwrite with no prompt) isn't needed since we tested target
           // didn't exist already
           component.addElevatedOperation("Execute", "cmd", "/C", "copy", sourceFile, targetFile, "/Y");
-          // Register it: Only for "OCX"
-          // If it's a .ocx (case insensitive), we save it to be registered
-          if (systemArray[i].toLowerCase().indexOf(".ocx") !== -1) {
-            dllsToReg.push(targetFile);
-          }
+        }
+        // Register it: Only for "OCX"
+        // On some systems these files may be present but not properly registered, so always register here
+        // If it's a .ocx (case insensitive), we save it to be registered
+        if (systemArray[i].toLowerCase().indexOf(".ocx") !== -1) {
+          dllsToReg.push(targetFile);
```

Comment:
Moved this out of the `if (!installer.fileExists(targetFile)) {` block so that the ocx files are always registered, even if they already were present.

Reply:
This is a great change.